src/main/webapp/app/shared/chart/active-students-chart.directive.ts

```diff
@@ -0,0 +1,66 @@
+import { Directive } from '@angular/core';
+import { Course } from 'app/entities/course.model';
+import dayjs from 'dayjs/esm';
+
+@Directive()
+export class ActiveStudentsChartDirective {
```

Comment:
Does this actually need to be a directive or could it be a regular class?

Reply:
Indeed, should be fixed